tests/conftest.py

```diff
@@ -0,0 +1,309 @@
+    <testsuite errors="0" failures="0" name="pytest" skips="0" tests="5" time="1.664">
+        <properties>
+            <property name="JENKINS_CONSOLE_LOG_URL" value="Unknown"/>
+            <property name="SCENARIO" value="Unknown"/>
+            <property name="ACTION" value="Unknown"/>
+            <property name="IMAGE" value="Unknown"/>
+            <property name="OS_ARTIFACT_SHA" value="Unknown"/>
+            <property name="PYTHON_ARTIFACT_SHA" value="Unknown"/>
+            <property name="APT_ARTIFACT_SHA" value="Unknown"/>
+            <property name="GIT_REPO" value="Unknown"/>
+            <property name="GIT_BRANCH" value="Unknown"/>
+        </properties>
+        <testcase classname="tests.test_default" file="tests/test_default.py" line="24" name="test_skip[ansible://localhost]" time="0.00197100639343">
+            <skipped message="unconditional skip" type="pytest.skip">tests/test_default.py:24: &lt;py._xmlgen.raw object at 0x7f0921ff4d50&gt;</skipped>
+        </testcase>
+    </testsuite>
+    """
+
+    with open(filename, 'w') as f:
+        f.write(junit_xml)
+
+    return filename
+
+
+@pytest.fixture(scope='session')
+def flat_all_passing_xml(tmpdir_factory):
+    """JUnitXML sample representing all passing tests."""
+
+    filename = tmpdir_factory.mktemp('data').join('flat_all_passing.xml').strpath
+    junit_xml = \
+    """<?xml version="1.0" encoding="utf-8"?>
+    <testsuite errors="0" failures="0" name="pytest" skips="0" tests="5" time="1.664">
+        <properties>
+            <property name="JENKINS_CONSOLE_LOG_URL" value="Unknown"/>
+            <property name="SCENARIO" value="Unknown"/>
+            <property name="ACTION" value="Unknown"/>
+            <property name="IMAGE" value="Unknown"/>
+            <property name="OS_ARTIFACT_SHA" value="Unknown"/>
+            <property name="PYTHON_ARTIFACT_SHA" value="Unknown"/>
+            <property name="APT_ARTIFACT_SHA" value="Unknown"/>
+            <property name="GIT_REPO" value="Unknown"/>
+            <property name="GIT_BRANCH" value="Unknown"/>
+        </properties>
+        <testcase classname="tests.test_default" file="tests/test_default.py" line="8" name="test_pass1[ansible://localhost]" time="0.00372695922852"/>
+        <testcase classname="tests.test_default" file="tests/test_default.py" line="12" name="test_pass2[ansible://localhost]" time="0.00341415405273"/>
+        <testcase classname="tests.test_default" file="tests/test_default.py" line="15" name="test_pass3[ansible://localhost]" time="0.00363945960999"/>
+        <testcase classname="tests.test_default" file="tests/test_default.py" line="18" name="test_pass4[ansible://localhost]" time="0.00314617156982"/>
+        <testcase classname="tests.test_default" file="tests/test_default.py" line="21" name="test_pass5[ansible://localhost]" time="0.00332307815552"/>
+    </testsuite>
+    """
+
+    with open(filename, 'w') as f:
+        f.write(junit_xml)
+
+    return filename
+
```

Comment:
need an extra space here?

Reply:
Yeah, it is too follow the pep8 styling standard.